core/src/main/scala/org/apache/gearpump/cluster/master/Master.scala

```diff
@@ -162,4 +166,6 @@ private[cluster] class Master extends Actor with Stash {
 
 object Master{
   case class WorkerTerminated(workerId : Int)
+
+  case class MasterInfo(master: ActorRef, startTime : Long)
```

Comment:
Is it make sense to add a default value for MasterInfo?

Like MasterInfo(master: ActorRef, startTime = 0L)